diff -urN nss-3.12.4.orig/mozilla/security/coreconf/UNIX.mk nss-3.12.4/mozilla/security/coreconf/UNIX.mk
--- nss-3.12.4.orig/mozilla/security/coreconf/UNIX.mk	2008-10-03 21:12:41.000000000 +0200
+++ nss-3.12.4/mozilla/security/coreconf/UNIX.mk	2009-09-26 01:11:29.595437017 +0200
@@ -51,13 +51,7 @@
 	DEFINES    += -DDEBUG -UNDEBUG -DDEBUG_$(USERNAME)
 endif
 
-ifdef BUILD_TREE
-NSINSTALL_DIR  = $(BUILD_TREE)/nss
-NSINSTALL      = $(BUILD_TREE)/nss/nsinstall
-else
-NSINSTALL_DIR  = $(CORE_DEPTH)/coreconf/nsinstall
-NSINSTALL      = $(NSINSTALL_DIR)/$(OBJDIR_NAME)/nsinstall
-endif
+NSINSTALL = $(CORE_DEPTH)/nsinstall
 
 MKDEPEND_DIR    = $(CORE_DEPTH)/coreconf/mkdepend
 MKDEPEND        = $(MKDEPEND_DIR)/$(OBJDIR_NAME)/mkdepend
