diff -urN nss-3.12.4.orig/mozilla/security/nss/lib/freebl/stubs.c nss-3.12.4/mozilla/security/nss/lib/freebl/stubs.c
--- nss-3.12.4.orig/mozilla/security/nss/lib/freebl/stubs.c	2009-09-26 15:15:49.950767715 +0200
+++ nss-3.12.4/mozilla/security/nss/lib/freebl/stubs.c	2009-09-26 15:17:29.860742297 +0200
@@ -529,8 +529,14 @@
 /*
  * fetch the library if it's loaded. For NSS it should already be loaded
  */
+#ifdef RTLD_NOLOAD
 #define freebl_getLibrary(libName)  \
     dlopen (libName, RTLD_LAZY|RTLD_NOLOAD)
+#else
+/* hope this doesn't break something */
+#define freebl_getLibrary(libName)  \
+    dlopen (libName, RTLD_LAZY)
+#endif
 
 #define freebl_releaseLibrary(lib) \
     if (lib) dlclose(lib)
